sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasQueryParameters.cs

```diff
@@ -129,7 +134,12 @@ internal DataLakeSasQueryParameters(UriQueryParamsCollection values)
         /// <returns>
         /// A URL encoded query string representing the SAS.
         /// </returns>
-        public override string ToString() =>
-            Encode(includeBlobParameters: true);
+        public override string ToString()
+        {
+            StringBuilder sb = new StringBuilder();
+            _keyProperties.AppendProperties(sb);
+            this.BuildParameterString(sb);
+            return sb.ToString();
```

Comment:
Nit - should we make these two names `AppendProperties` and `BuildParameterString` match?

Reply:
I can do AppendProperties for both.